analyzers/tests/SonarAnalyzer.UnitTest/TestCases/Hotspots/ExecutingSqlQueries.EF6.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Data.Entity;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Tests.Diagnostics
```

Comment:
Namespace are not needed (in most of the cases), they don't affect the rule content